_data/simple-icons.json

```diff
@@ -2819,6 +2819,16 @@
             "hex": "EF4223",
             "source": "https://www.codeigniter.com/help/legal"
         },
+        {
+            "title": "Codeium",
+            "hex": "09B6A2",
+            "source": "https://exafunction.github.io/simple-logo.svg",
```

Comment:
Where is this file in the source code? Ideally we should point to the file in a commit - as opposed to a GitHub.io address.